client/src/App.js

```diff
@@ -38,13 +38,6 @@ const httpLink = createHttpLink({
   uri: "/graphql",
 });
 
-//dark theme by default
```

Comment:
Need to add theme

Reply:
I have branches with the dark theme, so I will approve this, add the default back then do another pull request.